packages/dart/lib/src/objects/response/parse_error_response.dart

```diff
@@ -5,7 +5,8 @@ ParseResponse buildErrorResponse(
     ParseResponse response, ParseNetworkResponse apiResponse) {
   final Map<String, dynamic> responseData = json.decode(apiResponse.data);
   response.error = ParseError(
-      code: responseData[keyCode], message: responseData[keyError].toString());
-  response.statusCode = responseData[keyCode];
+      code: responseData[keyCode] ?? -1,
+      message: responseData[keyError].toString());
+  response.statusCode = responseData[keyCode] ?? -1;
```

Comment:
Do we have a list of Parse Errors in this SDK anywhere? These error codes are usually defined somewhere. See for example [ParseError](https://github.com/parse-community/Parse-SDK-JS/blob/alpha/src/ParseError.js) in the Parse JS SDK.

If the Flutter SDK doesn't have that, could you please centrally defined at least the `-1` error code with the same name as in the Parse JS SDK, and use that reference instead of directly returning `-1`.

Reply:
In the Flutter SDK, we have a list of errors in this [file](https://github.com/parse-community/Parse-SDK-Flutter/blob/d5f0657dd06fbb4ad6c7890303e946d72d39e773/packages/dart/lib/src/objects/parse_error.dart#L17), but the variable is private and cannot be used in other classes. I will try to improve this section.

Reply:
I think it should be public anyway, because a developer may want to reference the error code somewhere in their code to identify the error.

Reply:
> Amazing! Could you please preserve the numeric codes as well? It seems that in the commit they were removed.

all numeric codes are linked to const keys. like [this](https://github.com/parse-community/Parse-SDK-Flutter/blob/60a82288aa329b0537b5d936e6c5785837294be7/packages/dart/lib/src/objects/parse_error.dart#L228).
what codes do you mean ?

Reply:
Looking at it now it looks fine, I think I only saw a truncated version when looking at the changes previously.